frontend/scripts/react-components/shared/tags-group/tags-group.component.jsx

```diff
@@ -7,29 +7,43 @@ import { translateText } from 'utils/transifex';
 import 'react-components/shared/tags-group/tags-group.scss';
 
 function TagsGroup(props) {
-  const { as, tags, color } = props;
+  const { as, tags, color, suffix } = props;
   return React.createElement(
     as,
     { className: 'c-tags-group' },
-    tags.map(part => (
-      <Heading
-        as="span"
-        size="md"
-        align="center"
-        key={part.id}
-        color={color}
-        className="tag-group-part notranslate"
-      >
-        {part.prefix && translateText(`${part.prefix} `)}
-        {part.value && <Tag {...props} part={part} />}
-      </Heading>
-    ))
+    <>
+      {tags.map(part => (
+        <Heading
+          as="span"
+          size="md"
+          align="center"
+          key={part.id}
+          color={color}
+          className="tag-group-part notranslate"
+        >
+          {part.prefix && translateText(`${part.prefix} `)}
+          {part.value && <Tag {...props} part={part} />}
+        </Heading>
+      ))}
+      {suffix && tags.length > 0 && (
```

Comment:
It's confusing to have `part.prefix` and `suffix`. I think this could be achieved by adding a new part that only has a value `profile`